packages/stencil/src/components/datacom-toggle/datacom-toggle.tsx

```diff
@@ -0,0 +1,90 @@
+import { Component, h, Prop, Host, Event, EventEmitter } from '@stencil/core';
+
+export type ToggleSizeType = 'standard' | 'small';
+
+@Component({
+  tag: 'datacom-toggle',
+  styleUrl: 'datacom-toggle.css',
+  shadow: true,
+})
+export class DatacomToggle {
+  /**
+   * Label of the toggle element
+   */
+  @Prop() label;
+
+  /**
+   * True if label is on the right of element
+   */
+  @Prop() labelOnLeft = false;
```

Comment:
I think this should be called labelPosition with values of 'left' | 'right' with a default of right

Reply:
Just made the change. Let me know if the change is how you envisioned.